app/src/main/java/org/oppia/android/app/options/AppLanguageActivityPresenter.kt

```diff
@@ -5,40 +5,41 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.databinding.DataBindingUtil
 import org.oppia.android.R
 import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.app.model.OppiaLanguage
 import org.oppia.android.databinding.AppLanguageActivityBinding
 import javax.inject.Inject
 
 /** The presenter for [AppLanguageActivity]. */
 @ActivityScope
 class AppLanguageActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
-  private lateinit var prefSummaryValue: String
+  private lateinit var oppiaLangauge: OppiaLanguage
 
-  fun handleOnCreate(prefKey: String, prefValue: String) {
+  fun handleOnCreate(prefValue: OppiaLanguage) {
```

Comment:
Nit: please rename ``prefValue`` to something more consistent.

Reply:
Updated.